### PR TITLE
Small edits across docs and tutorials

### LINKED ISSUE #30: Tutorial in the docs seems outdated
https://trixi-framework.github.io/HOHQMesh.jl/dev/guided-tour/#Build-a-mesh-from-scratch

In particular, `saveProject` does not run automatically when `generate_mesh` is called, and `saveProject(p, ::String)` doesn't seem to work (only a `saveProject(p)` signature appears to be exported). This is with v0.2.1 on @marklopat's Windows machine.

### PR DESCRIPTION
**Pull request by andrewwinters5000**
Closes #30 

Also, because the `fmin` function and some other functions in HOHQMesh were re-written the mesh statistics change (very) slightly such that the reference values in the tutorials are updated.

### CODE REVIEW

File: docs/src/development.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ are available for almost any operating system. They also are preinstalled on
 virtually all Unix-like systems.  However, Vim and Emacs come with their own,
 steep learning curve if they have never been used before. Therefore, if in doubt, it
 is probably easier to get started with a classic GUI-based text editor (like
-Juno). If you decide to use Vim or Emacs, make sure that you install the
+VS Code). If you decide to use Vim or Emacs, make sure that you install the
```
Comment (ranocha) [inline docs/src/development.md]:
```suggestion
VS Code with the [Julia extension](https://code.visualstudio.com/docs/languages/julia)).
If you decide to use Vim or Emacs, make sure that you install the
```

File: docs/src/guided-tour.md [hunk 2]
```diff
@@ -132,9 +132,10 @@ Finally, the script returns the project so that it can be edited further, if des
 
 To save a control file for HOHQMesh, simply invoke
 ```
-saveProject(proj::Project, outFile::String)
+saveProject(proj::Project)
 ```
-where `outFile` is the name of the control file (traditionally with a .control extension).
+where the name of the control file (traditionally with a .control extension) matches the
```
Comment (ranocha) [inline docs/src/guided-tour.md]:
```suggestion
where the name of the control file (traditionally with a `.control` extension) matches the
```

File: docs/src/guided-tour.md [hunk 3]
```diff
@@ -132,9 +132,10 @@ Finally, the script returns the project so that it can be edited further, if des
...
+name of the `proj`.
```
Comment (ranocha) [inline docs/src/guided-tour.md]:
```suggestion
name of the project `proj`.
```

File: docs/src/interactive-api.md [hunk 4]
```diff
@@ -106,10 +107,12 @@ By default, the mesh, plot and stats files will be written with the name and pat
...
-A default smoother is created when `newProject` is called, which sets the status to `ON`, type to
-`LinearAndCrossbarSpring`, and number of iterations = 25. These are generally good enough for most purposes.
-The most likely parameter to change is the number of iterations. Further details on the smoothing strategy
-and how it works are available [here](https://trixi-framework.github.io/HOHQMesh/the-control-input/#the-smoother).
+A default smoother is created when `newProject` is called, which sets the status to `ON`,
+type to `LinearAndCrossbarSpring`, and number of iterations = 25.
+These are generally good enough for most purposes.
+The most likely parameter to change is the number of iterations.
+Further details on the smoothing strategy and how it works are available
+[here](https://trixi-framework.github.io/HOHQMesh/the-control-input/#the-smoother).
```
Comment (ranocha) [inline docs/src/interactive-api.md]:
```suggestion
[in the documentation](https://trixi-framework.github.io/HOHQMesh/the-control-input/#the-smoother).
```

**Review by ranocha — COMMENTED** [review]
Thanks! I just have a few minor suggestions

**Comment by andrewwinters5000** [comment]
@ranocha Are the other checks not running because only the documentation source changed?

**Comment by ranocha** [comment]
Yes

**Pull request merged**